.github/workflows/avm.res.desktop-virtualization.workspace.yml

```diff
@@ -0,0 +1,83 @@
+name: "avm.res.desktop-virtualization.workspace"
+
```

Comment:
Hey @moisesjgomez, don't forget the change to the codeowners file as suggested in other PRs

Reply:
Codeowners file was update along with creation of Github teams.